app/views/categories/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="">
+<div class="bg-primary py-4">
+<div class="container">
+  <div class="row">
+  <div class="col">
+   <button class="navbar-toggler ms-auto text-white" type="button" data-mdb-toggle="collapse"
+      data-mdb-target="#navbarToggleExternalContent3" aria-controls="navbarToggleExternalContent3"
+      aria-expanded="true" aria-label="Toggle navigation">
+      <i class="fas fa-bars"></i>
+    </button>
+    </div>
+    <div class="col">
+    <h1 class="text-white">TRANSACTIONS</h1>
+    </div>
+    <div class="col justify-content-end d-flex">
+    <button id="search-button" type="button" class="btn btn-primary">
+    <i class="fa fa-search text"></i>
+  	</button>
+    </div>
+  </div>
+  </div>
+  <div class="container d-flex justify-content-end gap-3 mt-3 mb-3">
+   
+    <button>Most Recent</button>
+    <button>Most Ancient</button>
+  </div>
+  </div>
+  <div id="categories" class="container bg-light">
+    <% @categories.each do |category| %>
+      <%= link_to "/categories/#{category.id}", class: "text-decoration-none" do %>
+        <div class="container bg-white mt-4 shadow-lg p-3 mb-5 rounded">
+          <div class="row">
+            <div id="img-div" class="col"><img src=<%= category.icon%> alt=<%= category.name%> /></div>
+            <div class="col">
+              <p class = "fw-bold text-secondary fs-5"><%= category.name %><p>
+              <p class = "fw-light text-secondary"><%= category.created_at.to_date.strftime('%Y %b %d') %></p>
+            </div>
+            <div class="col d-flex justify-content-end">
+              <p class = "fw-bold text-dark fs-5">$<%= category.count_total_payments %></p>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+  <div class="mt-3 mb-5 d-flex justify-content-center">
+    <%= link_to "New category", new_category_path, class: "btn btn-primary" %>
+  </div>
+</div>
```

Comment:
- [ ] Kindly consider following the design guideline as shown in [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) by following the same color and design in all screens.